bbb-install-2.6.sh

```diff
@@ -1012,24 +1074,28 @@ fingerprint
 lt-cred-mech
 use-auth-secret
 static-auth-secret=$COTURN_SECRET
-realm=$(echo "$COTURN_HOST" | cut -d'.' -f2-)
-
-cert=/etc/turnserver/fullchain.pem
-pkey=/etc/turnserver/privkey.pem
-# From https://ssl-config.mozilla.org/ Intermediate, openssl 1.1.0g, 2020-01
-cipher-list="ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384"
-dh-file=/etc/turnserver/dhp.pem
+realm=$HOST
 
 keep-address-family
 
 no-cli
 no-tlsv1
 no-tlsv1_1
-HERE
 
-  mkdir -p /etc/turnserver
-  if [ ! -f /etc/turnserver/dhp.pem ]; then
-    openssl dhparam -dsaparam  -out /etc/turnserver/dhp.pem 2048
+# Block connections to IP ranges which shouldn't be reachable
+no-loopback-peers
+no-multicast-peers
+
+
+# we only need to allow peer connections from the machine itself (from mediasoup or freeswitch).
+denied-peer-ip=0.0.0.0-255.255.255.255
+denied-peer-ip=::-ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff
+allowed-peer-ip=$IP
+
+HERE
+  else
+    # fetch secret for later setting up in BBB turn config
+    COTURN_SECRET="$(grep static-auth-secret= /etc/turnserver.conf |cut -d = -f 2)"
```

Comment:
here a trainling "=" will be ommitted. (and that is *always* there, due to `openssl rand -base64 32` previously)
correct variant:
`COTURN_SECRET="$(grep static-auth-secret= /etc/turnserver.conf |cut -d = -f 2-)"`